server/plugin.go

```diff
@@ -142,6 +151,23 @@ func signClaims(secret string, claims *Claims) (string, error) {
 	return string(token.Raw()), nil
 }
 
+func (p *Plugin) trackMeeting(args *model.CommandArgs) {
+	// disables tracking if the user is not using the default jitsi url
+	isNotDefaultJitsiURL := p.isNotDefaultJitsiURL()
+	// enables tracking based on the users configuration
+	var event = map[string]interface{}{
+		"external-meeting-link-enabled": isNotDefaultJitsiURL,
+	}
+
+	_ = p.tracker.TrackEvent("meeting-link", event)
```

Comment:
Let's give this a more self describing name
```suggestion
	_ = p.tracker.TrackEvent("start_meeting", event)
```